environment.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: pudl-archiver
+name: pudl-cataloger
```

Comment:
This environment name change fixes a super weird bug. If you think of a better name than `pudl-cataloger` then feel free to change.

Reply:
What's the bug Katie? I changed this back and it works just fine in my environment.

Reply:
It's an bug with python on MacOS where if `-arch` is in the name of the environment then it is interpreted as the `-arch` architecture flag. So I think for the sake of this environment working on MacOS this environment name should be changed.

Reply:
Got it - would it be neater to make this change in one PR for all the archivers, or has this been done already?